src/main/java/com/axway/apim/swagger/APIImportConfigAdapter.java

```diff
@@ -109,6 +113,18 @@ public IAPI getDesiredAPI() throws AppException {
 			} else {
 				stagedConfig = baseConfig;
 			}
+			Path currentRelativePath = Paths.get("");

```

Comment:
Please externalize this code into a method.

Reply:
ok

Reply:
I meant the whole section, basically starting from:
```
if(getStageConfig(stage, apiConfig)!=null) {
	LOG.info("Overriding configuration from: " + getStageConfig(stage, apiConfig));
	stagedConfig = updater.readValue(new File(getStageConfig(stage, apiConfig)));
} else {
	stagedConfig = baseConfig;
}
```
All the new code until:
```
addDefaultPassthroughSecurityProfile(stagedConfig);
```

Reply:
Thx!